backend/migrations/20230703130438_food.down.sql

```diff
@@ -1,4 +1,4 @@
 -- Add down migration script here
 DROP TABLE food;
 
-DROP TYPE meal_type;
```

Comment:
don't change old migration, create new one where you rename

Reply:
Is this not the original name? I thought i had changed it back